tla/download_or_update_tla.sh

```diff
@@ -39,7 +39,7 @@ print_version() {
 main() {
 	echo "Downloading tla2tools.jar..."
 	before=$(date -r tla2tools.jar 2>/dev/null)
-	download https://tla.msr-inria.inria.fr/tlatoolbox/dist/tla2tools.jar
+	download https://github.com/tlaplus/tlaplus/releases/latest/download/tla2tools.jar
```

Comment:
So, there is this comment in the current TLA+ README.md
https://github.com/microsoft/CCF/blob/f357a5e3a3aab77c61a7fce7a5b2ced558e0103e/tla/README.md?plain=1#L15

Is there somewhere else which you think might be better to put this information? I was separately considering moving the content of this readme (and the others in readmes in the TLA+ directory) into the TLA+ docs page so there's a single source of information (and for consistency with the rest of the CCF repo).

Yep. I believe @fritzalder used v1.7.1. I am currently running v2.17 (the latest version) but I have not yet updated the comment about the version number since I have not finished checking the current specs (see https://github.com/microsoft/CCF/issues/3837 & https://github.com/microsoft/CCF/issues/3828)

Otherwise, I could pin the URL to the latest version if you think that might be a better approach?

Reply:
I think either is fine in this case, as long as it's easy to get to a version of `tlaplus` that works as we don't run these tests often.